scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -14,3 +14,6 @@ export CAPI_REL_TO_VERSION="v0.3.14"
 
 export CAPI_API_VERSION="v1alpha3"
 export CAPM3_API_VERSION="v1alpha4"
+
+export KUBERNETES_VERSION="v1.21.1"
```

Comment:
this should be 1.21.0 otherwise the upgrade will fail.

Reply:
This PR is on old. I have create a new one https://github.com/metal3-io/metal3-dev-env/pull/670